src/vs/workbench/contrib/notebook/browser/services/notebookServiceImpl.ts

```diff
@@ -490,6 +493,35 @@ export class NotebookService extends Disposable implements INotebookService {
 			this._onDidChangeOutputRenderers.fire();
 		});
 
+		notebookPreloadExtensionPoint.setHandler(extensions => {
+			this._notebookStaticPreloadInfoStore.clear();
```

Comment:
@mjbvz 
I've tried this branch and couldn't get this to work.
Possible because we have `setHandler` here and the fact that this could be invokved after `getStaticPreloads` is called. Meaning we might need a way to trigger an event indicating there are changes (contributions) to the `pre-Loads`

Reply:
@DonJayamanne I think that would only effect notebooks that are opened on boot. Can you share the contribution you tried? Make sure you also enable this proposed API. Otherwise the contribution will be ignored

Reply:
> Make sure you also enable this proposed API

Ooops missed that, that's what I was missing